mobile/src/main/java/de/michelinside/glucodatahandler/widget/GlucoseBaseWidget.kt

```diff
@@ -28,15 +30,17 @@ enum class WidgetType(val cls: Class<*>) {
     GLUCOSE_TREND_DELTA(GlucoseTrendDeltaWidget::class.java),
     GLUCOSE_TREND_DELTA_TIME(GlucoseTrendDeltaTimeWidget::class.java),
     GLUCOSE_TREND_DELTA_TIME_IOB_COB(GlucoseTrendDeltaTimeIobCobWidget::class.java),
-    OTHER_UNIT(OtherUnitWidget::class.java);
+    OTHER_UNIT(OtherUnitWidget::class.java),
+    BATTERY_LEVEL(BatteryLevelWidget::class.java);
```

Comment:
I think you can not use this base class, as these widget will not be updated for Battery-Level change.
You have to use an own class with parent NotifierInterface and register for NotifySource.NODE_BATTERY_LEVEL.

Reply:
Also for NotifySource.CAPILITY_INFO, this will be called, if wear is connected or disconnected...

The widget Notifier should only be active, if there is at least one Widget active.
The object/class must be initialized via GlucoDataServiceMobile::onCreate(): https://github.com/pachi81/GlucoDataHandler/blob/f546973c70d8fbbccfaf89e3b0e3111f792d8652/mobile/src/main/java/de/michelinside/glucodatahandler/GlucoDataServiceMobile.kt#L165

Reply:
I'll investigate this